### ISSUE TITLE
Want to generate Instagram styled captions 

### ISSUE THREAD

**Issue by shreyassks** [232525/PureT#17]
Hi, I just wanted to know if this model architecture works well to generate instagram styled captions??. I am using YFCC100m dataset which has image-caption pairs which serve for my purpose. But when i use this architecture and train it, I don't see the model is able to learn anything. I have used CE loss and am yet to try SCST training. Please suggest

**Comment by 232525** [comment]
I am sorry that I have no idea about the performance under YFCC100m dataset.  But I used to train our model under a Chinese Caption dataset and it get normal performance. And I guess maybe the codes about Dataset and Dataloader are important, so maybe you need to check them to make sure they work correctly.

**Comment by shreyassks** [comment]
Thanks for your response. Unlike COCO dataset which has 5 captions per image, in YFCC there is just one per image. By keeping the hyperparams anc architecture the same as PureT, Beam search was just generating <SOS> tokens. So I was curious if there was any way to achieve similar performance with little tweaks for YFCC
